src/khoj/utils/helpers.py

```diff
@@ -214,13 +214,15 @@ def log_telemetry(
 
 
 class ConversationCommand(str, Enum):
+    Default = "default"
     General = "general"
     Notes = "notes"
     Help = "help"
 
 
 command_descriptions = {
-    ConversationCommand.General: "This command allows you to search talk with the LLM without including context from your knowledge base.",
-    ConversationCommand.Notes: "This command allows you to search talk with the LLM while including context from your knowledge base.",
-    ConversationCommand.Help: "This command displays a help message with all available commands and other metadata.",
+    ConversationCommand.General: "Only talk about information that relies on Khoj's general knowledge, not your personal knowledge base.",
```

Comment:
In edffbad you refer to "my" notes, here you refer to khoj's knowledge base. If this misalignment in ownership context significant? Or should it be Claridge that my notes are part of your knowledge badge?

Reply:
@nickanderson , thanks for pointing this out -- I think this might be way Llama sort of loses the thread sometimes and messed up its person. I do think this is significant; good catch!